src/ussd_covid19_triage.js

```diff
@@ -255,6 +428,36 @@ go.app = (function () {
       });
     });
 
+    self.add("state_age_years", function (name) {
+      if (self.im.user.answers.state_age_years && self.im.user.answers.state_age) {
+        return self.states.create("state_province");
+      }
+      var question = $("Please TYPE your age in years (eg. 35)");
+      return new FreeText(name, {
+        question: question,
+        check: function(content) {
+          var age = _.parseInt(content);
+          if(age < 1) {
+            return question;
+          } else if (age < 18) {
+            self.im.user.answers.state_age = "<18";
+            return;
+          } else if (age < 40) {
+            self.im.user.answers.state_age = "18-40";
+            return;
+          } else if (age <= 65) {
+            self.im.user.answers.state_age = "40-65";
+            return;
+          } else if (age < 150) {
+            self.im.user.answers.state_age = "40-65";
+            return;
+          }
```

Comment:
You could use a switch statement here

Reply:
I'm not sure how I could use a switch statement without it being very verbose? Switch statements have to match on exact results, so I'm not sure how to do it besides doing something like:
```
switch(age) {
  case 1:
  case 2:
  case 3:
  case 4:
  ...
    self.im.user.answers.state_age = "<18";
    break;

  case 18:
  case 19:
  ...
}
```

Reply:
switch(age) {
  case age < 150:
    self.im.user.answers.state_age = "<18";
    break;
  ...
}

Reply:
Yeah that doesn't work for me:
```
> var age = 23;
> var result = null;
> switch(age) {
... case age < 18:
...   result = "<18";
...   break;
... case age < 40:
...   result = "18-40";
...   break;
... }
> result
null
```
And I don't see that being possible anywhere in the docs: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch

Reply:
https://www.w3schools.com/js/tryit.asp?filename=tryjs_switch, conditional case statements are acceptable

Reply:
That link is still a `case 0:` `case 1:`, etc. I don't see any `case age < 18:`, and it doesn't work for me when I try it

Reply:
@rudigiesler not a train smash, but this does make sense in a switch statement